File: SayidanYaziyaDonusturme/SayidanYaziyaDonusturme.java
package sayidanyaziyadonusturme;

import java.util.Scanner;

public class SayidanYaziyaDonusturme {

    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.print("1-99 arası bir sayı giriniz: ");
        int sayi = reader.nextInt();
        String[] birler = {"Bir", "İki", "Üç", "Dört", "Beş", "Altı", "Yedi", "Sekiz", "Dokuz"};
        String[] onlar = {"On", "Yirmi", "Otuz", "Kırk", "Elli", "Altmış", "Yetmiş", "Seksen", "Doksan"};
        int onlarbasamagi = sayi / 10;
        int birlerbasamagi = sayi % 10;
        for (int i = 0; i < 9; i++) {
            if (onlarbasamagi == i + 1) {
                System.out.print(onlar[i] + " ");
            }
        }
        for (int i = 0; i <= 8; i++) {
            if (birlerbasamagi == i + 1) {
                System.out.println(birler[i]);
            }
        }
    }
}

File: SayidanYaziyaDonusturme/SayidanYaziyaDonusturme.py
def sayidan_yaziya_donusturme():

    birler = ["Bir", "İki", "Üç", "Dört", "Beş", "Altı", "Yedi", "Sekiz", "Dokuz"]
    onlar = ["On", "Yirmi", "Otuz", "Kırk", "Elli", "Altmış", "Yetmiş", "Seksen", "Doksan"]
    sayi = int(input("1-99 arası bir sayı giriniz : "))
    onlar_basamagi = int(sayi / 10)
    birler_basamagi = int(sayi % 10)
    for i in range(9):
        if onlar_basamagi == i+1:
            print(onlar[i], end=' ')

    for i in range(9):
        if birler_basamagi == i+1:
            print(birler[i])

sayidan_yaziya_donusturme()